SHOWRAW.BAS

```diff
@@ -1,32 +1,116 @@
-REM A script to show the raw screen data saved in MANDELBR.BAS
-
 DECLARE SUB LoadRawScreen (filename AS STRING, w AS INTEGER, h AS INTEGER)
-
+DECLARE SUB CyclePalette ()
 SCREEN 13
 CLS
-LINE(0,0)-(320-200), 15, BF
+LINE (0, 0)-(320, 200), 15, BF
+CALL LoadRawScreen("C:\SCREEN.RAW", 320, 200)
```

Comment:
This deviates from the existing code in MANDELBR.BAS [here](https://github.com/adamlwgriffiths/qbasic-fractal/blob/master/MANDELBR.BAS#L46),
which outputs the file to "mandelbr.raw".
These should be synced up.

Reply:
Fixed.